.gitsubrepo

```diff
@@ -0,0 +1,4 @@
+[ remote: https://github.com/ ]
+[ base: ]
+
+k3mime/thirdpart/mimes.json    bsc-s2/resource             master    mimes.json
```

Comment:
I think it is needed to check whether this config works

Reply:
I have checked and this config works.

Reply:
I change "k3mime/thirdpart/mimes.json" to "thirdpart/mimes.json"